docs/reference-guides/rancher-webhook.md

```diff
@@ -17,6 +17,26 @@ Rancher-Webhook is crucial for Rancher to protect clusters against malicious att
 Rancher relies on the Rancher-Webhook as an integral part of its functionality. Without the webhook, Rancher would not be a complete product. 
 It provides essential protection for Rancher-managed clusters, preventing security vulnerabilities and ensuring the consistency and stability of the cluster.
 
+## What resources does the webhook validate?
+
+An in-progress list of the resources that the webhook validates can be found in the [webhook's repo](https://github.com/rancher/webhook/blob/release/v0.4/docs.md). Checks specific to one version can be found by viewing the `docs.md` file associated with a particular tag (note that webhook versions prior to `v0.3.6` won't have a version).
+
+## Bypassing the webhook
+
+In certain use cases, it may be necessary to bypass the validation that the webhook provides in order to perform emergency restore operations, or fix other critical issues. The bypass operation is exhaustive, meaning that no checks of the webhook will apply when this is used. It is not possible to bypass some checks and have others still apply - they are either all bypassed, or all active. 
```

Comment:
```suggestion
Sometimes it may be necessary to bypass webhook validation to perform emergency restore operations, or fix other critical issues. The bypass operation is exhaustive, meaning that no webhook checks will apply when this is used. It is not possible to bypass some checks and have others still apply - they are either all bypassed, or all active. 
```